RegisterPayments/index.ts

```diff
@@ -0,0 +1,56 @@
+import { Context } from "@azure/functions";
+
+import * as express from "express";
+import * as winston from "winston";
+
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+import {
+  getRTBlobStorageService,
+  IBlobStorageParams
+} from "../utils/blobStorage";
+import { IBasicAuthParams } from "../utils/middlewares/authorization";
+import { RegisterPayment } from "./handler";
+
+const basicAuthParams: IBasicAuthParams = {
+  clientId: getRequiredStringEnv("AUTH_CLIENT_ID"),
+  secret: getRequiredStringEnv("AUTH_CLIENT_SECRET")
+};
+
+const rtBlobStorageParams: IBlobStorageParams = {
+  connectionString: getRequiredStringEnv("RTStorageConnection"),
+  containerName: getRequiredStringEnv("RTContainerName")
+};
+
+const rtBlobStorageService = getRTBlobStorageService(rtBlobStorageParams);
```

Comment:
`getRTBlobStorageService` may throw

Reply:
`getRTBlobStorageService` cannot throw. `save` method inside it can. I'll handle the exception properly when `save` is used inside the handler